### PR TITLE
File transfer API

### PR DESCRIPTION
**Pull request by pun-ky**

### CODE REVIEW

File: aem/gradle/environment.gradle.kts [hunk 1]
```diff
@@ -33,23 +33,31 @@ configure<AemExtension> {
...
+            description = "Migrates pages to new component"
             doLast {
-                // ...
+                aem.anyInstance.sync {
+                    repository {
+                        node("/content/example").traverse()
+                                .filter { it.type == "cq:PageContent" && properties["sling:resourceType"] == "example/components/basicPage" }
+                                .forEach { page ->
+                                    logger.info("Migrating page: ${page.path}")
+                                    page.saveProperty("sling:resourceType", "example/components/advancedPage")
```
Comment (mierzwid) [inline aem/gradle/environment.gradle.kts]: LIKE

File: gradle/fork/gradle.properties.peb [hunk 2]
```diff
@@ -16,9 +16,9 @@ instance.local-publish.jvmOpts={{aemInstanceJvmOpts}}
 {% endif %}
 
 {% if aemInstanceType == 'local' %}
-localInstance.zipUrl={{aemLocalInstanceZipUri}}
-localInstance.jarUrl={{aemLocalInstanceJarUri}}
-localInstance.licenseUrl={{aemLocalInstanceLicenseUri}}
+localInstance.backup.downloadUrl={{aemLocalInstanceZipUri}}
```
Comment (mierzwid) [inline gradle/fork/gradle.properties.peb]: So backup it will have prefix localInstance?

**Comment by mierzwid** [comment]
Is it already migrated to GAP 7.0.0?

File: gradle/fork/gradle.properties.peb (hunk 2 above)
Comment (pun-ky) [reply gradle/fork/gradle.properties.peb]: yep, you cannot backup remotes ;)

and it is somehow paired with 

`localInstance.quickstart` and `localInstance.backup` ; looks nice in my opinion;

one thing for creating from scratch, second for restoring

**Comment by pun-ky** [comment]
> Is it already migrated to GAP 7.0.0?

yep

**Pull request merged**